mobile-app/lib/ui/views/podcast/widgets/podcast_title_widget.dart

```diff
@@ -185,214 +188,223 @@ class PodcastTileState extends State<PodcastTile> {
+      children: [
+        const SizedBox(height: 6),
+        Text(
+          firstSentence,
+          style: const TextStyle(
+            color: FccColors.gray10,
+            fontSize: 17,
           ),
-        );
-      },
-      minVerticalPadding: 16,
-      isThreeLine: true,
-      subtitle: Column(
-        children: [
-          descriptionWidget(),
-          footerWidget(context),
-        ],
+        ),
+        const SizedBox(height: 14),
+      ],
+    );
+  }
+
+  Widget _buildEpisodeDate() {
+    final date = widget.episode.publicationDate;
+    return Text(
+      date != null ? Jiffy.parseFromDateTime(date).fromNow().toUpperCase() : '',
+      style: const TextStyle(
+        fontSize: 13,
+        color: FccColors.gray10,
+        fontWeight: FontWeight.w600,
+        letterSpacing: 1.1,
       ),
     );
   }
 
-  Row footerWidget(BuildContext context) {
-    return Row(
-      children: [
-        (widget._audioService.playbackState.value.processingState ==
-                        AudioProcessingState.loading ||
-                    widget._audioService.playbackState.value.processingState ==
-                        AudioProcessingState.buffering) &&
-                widget._audioService.episodeId == widget.episode.id
-            ? Padding(
-                padding: const EdgeInsets.all(8.0),
+  Widget _buildPlayButton(bool isPlaying) {
+    return Container(
+      decoration: BoxDecoration(
+        color: isPlaying ? FccColors.purple90 : FccColors.gray80,
+        borderRadius: BorderRadius.circular(24),
+      ),
+      child: IconButton(
+        padding: const EdgeInsets.symmetric(horizontal: 18),
+        constraints: const BoxConstraints(),
+        icon: _isEpisodeLoading()
+            ? const Center(
                 child: SizedBox(
-                    height: MediaQuery.of(context).size.height * 0.0375,
-                    width: MediaQuery.of(context).size.height * 0.0375,
-                    child: const CircularProgressIndicator()),
+                  width: 28,
+                  height: 28,
+                  child: CircularProgressIndicator(
+                    color: FccColors.gray00,
+                    strokeWidth: 2.5,
+                  ),
+                ),
               )
-            : playbuttonWidget(context),
-        widget.isDownloading &&
-                widget._downloadService.downloadId == widget.episode.id
+            : Icon(
+                widget.playing ? Icons.pause : Icons.play_arrow,
+                color: FccColors.gray00,
+                semanticLabel:
+                    widget.playing ? 'Pause episode' : 'Play episode',
+              ),
+        onPressed: () {
+          playBtnClick();
+        },
+        iconSize: 28,
+      ),
+    );
+  }
+
+  Widget _buildDownloadButton() {
+    final isCurrentDownload = widget.isDownloading &&
+        widget._downloadService.downloadId == widget.episode.id;
+    return SizedBox(
```

Comment:
No need of an extra container. The stylings can be set directly on the button styles itself